src/test/java/eu/enmeshed/EnmeshedOnboardingServiceTest.java

```diff
@@ -197,44 +196,54 @@ void itShouldReturnRegistrationData() {
                 - Instant.now().toEpochMilli()
             > 3_595_000);
 
-    RelationshipTemplateContent.ItemList itemList =
+    Request request =
         relationshipTemplateCreationArgumentCaptor.getValue().getContent().getOnNewRelationship();
 
     // Shared Items
-    Assertions.assertTrue(itemList.items().get(0).getItems().get(0).getMustBeAccepted());
-    Assertions.assertEquals(testDisplayNameSharedAttributes, itemList.items().get(0).getTitle());
+    Assertions.assertTrue(
+        ((RequestItemGroup) request.getItems().get(0)).getItems().get(0).getMustBeAccepted());
```

Comment:
What does this `RequestItemGroup` do here? :D

Reply:
typecast (same as `as RequestItemGroup` in typescript)